geomesa-tools/src/main/scala/org/locationtech/geomesa/tools/export/ExportParams.scala

```diff
@@ -11,37 +11,41 @@ package org.locationtech.geomesa.tools.export
 import java.io.File
 
 import com.beust.jcommander.{Parameter, Parameters}
+import org.locationtech.geomesa.tools._
 import org.locationtech.geomesa.tools.{CatalogParam, OptionalCqlFilterParam, OptionalIndexParam, RequiredTypeNameParam}
 
-trait BaseExportParams extends CatalogParam with RequiredTypeNameParam with OptionalCqlFilterParam with OptionalIndexParam {
+trait RootExportParams extends OptionalCqlFilterParam {
   @Parameter(names = Array("-m", "--max-features"), description = "Maximum number of features to return. default: Unlimited")
   var maxFeatures: Integer = null
 
-  @Parameter(names = Array("-a", "--attributes"), description = "Attributes from feature to export " +
-      "(comma-separated)...Comma-separated expressions with each in the format " +
-      "attribute[=filter_function_expression]|derived-attribute=filter_function_expression. " +
-      "filter_function_expression is an expression of filter function applied to attributes, literals " +
-      "and filter functions, i.e. can be nested")
-  var attributes: java.util.List[String] = null
-
   @Parameter(names = Array("-o", "--output"), description = "Output to a file instead of std out")
   var file: File = null
 
   @Parameter(names = Array("--gzip"), description = "Level of gzip compression to apply to output, from 1-9")
   var gzip: Integer = null
-}
 
-// @Parameters(commandDescription = "Export features from a GeoMesa data store")
-trait ExportParams extends BaseExportParams {
-  @Parameter(names = Array("-F", "--format"), description = "Format to export (avro|bin|csv|geojson|gml|null|shp|tsv)")
-  var format: String = "csv"
+  @Parameter(names = Array("-F", "--output-Format"), description = "File format of output files (csv|tsv|gml|json|shp|avro)")
+  var outputFormat: String = "csv"
 
   @Parameter(names = Array("--no-header"), description = "Export as a delimited text format (csv|tsv) without a type header", required = false)
   var noHeader: Boolean = false
 }
 
-@Parameters(commandDescription = "Export features from a GeoMesa data store in a binary format")
-trait BinExportParams extends BaseExportParams {
+
+trait BaseExportParams extends RootExportParams
```

Comment:
i think our naming is getting a little muddy... what is the difference between 'base' and 'root' (rhetorical question)? maybe we could come up with something better